search-app/docs/api.md

```diff
@@ -5405,3 +5409,101 @@ Example un-healthy response body:
+ {"generic_association":{"concept_id":"GA1200000011-CMR","revision_id":1},"associated_item":{"concept_id":"V1200000008-PROV1","revision_id":1}}]
+ ```
+#### <a name="associations-in-search-result"></a> Associations in the search result 
+Latest revisions of associations can be queried by searching for one of the assocaited concepts using either the JSON or UMM_JSON format. The "associations" field, contains a list of concept ids associated to the concept that was searched for grouped by concept-type. The "association-details" field contains concept ids, as well as association details like revision ids and data payloads if they were included for the particular association. In the example case, a grid (GRD1200000000-PROV1) has been associated to another grid with association details(GRD1200000001-PROV1), as well as a service option (SO1200000001-PROV1) without any association details.
+Note: If you update an association with new data payload information, a new revision of that association will be created. Old revisions of the association will be preserved in the database but won't show up in the search result of the associated concepts.
+
+ Example
+
+ ```
+ curl -H "Authorization Bearer: XXXXX" "%CMR-ENDPOINT%/grids?concept_id=GRD1200000000-PROV1"
+
+ ```
+ Example Response with associated concepts
+
+ ```
+ {
+     "hits": 1,
+     "took": 6,
+     "items": [
+         {
+             "concept_id": "GRD1200000000-PROV1",
+             "revision_id": 22,
+             "provider_id": "PROV1",
+             "native_id": "sampleNativeId",
+             "name": "Grid-v1",
+             "associations": {
+                 "grids": [
+                     "GRD1200000001-PROV1"
+                 ],
+                 "serviceoptions": [
+                     "SO1200000001-PROV1"
+                 ]
+             },
+             "association-details": {
+                 "grids": [
+                     {
+                         "data": {
+                             "somedata": "zarr",
+                             "somemoredata": {
+                                 "XYZ": "ZYX"
+                             }
+                         },
+                         "concept-id": "GRD1200000001-PROV1",
+			 "revision-id": 1
+                     }
+                 ],
+                 "serviceoptions": [
+                     {
+                         "concept-id": "SO1200000001-PROV1"
+                     }
+                 ]
+             }
+         }
+     ]
+ }
+ ```
+#### <a name="concept-dissociations"></a> Concept dissociations 
+
+A concept, with optional revision id, can be dissociated from one or more other concepts, with optional revision ids
+
+ Example
+
+ ```
+ curl -XDELETE -H "Authorization: Bearer XXXXX" -H "Content-Type:application/json"
+      "%CMR-ENDPOINT%/associate/SE1200000006-PROV1/3"
+      -d '[{"concept_id": "V1200000008-PROV1", "revision_id": 1},
+           {"concept_id": "C1200000005-PROV1"}]'
```

Comment:
Do we support dissociation with collections? The example request and example response do not match.

Reply:
Yes we do. will double check the response. cut and pasted from somewhere else.